moneo.py

```diff
@@ -182,9 +320,9 @@ def check_insights_config(args, parser):
             check_deploy_shutdown(args, parser)
             mCLI.stop()
         elif args.job_id:
-            if (not os.path.isfile(args.host_ini)):
+            if (not os.path.isfile(args.host_file)):
                 print(
-                    args.host_ini + " does not exist. Please provide a host file. i.e. host.ini.\n")
+                    args.host_file + " does not exist. Please provide a host file. i.e. hostfile.\n")
```

Comment:
Should the comment say something about how to provide the hostfile?

Reply:
Ah, sorry my fault. I didn't notice print_help() immediately after this. That should be good enough.